webapp/i18n/ja.json

```diff
@@ -2012,6 +2054,7 @@
   "user.settings.security.ldap": "AD/LDAP",
   "user.settings.security.loginGitlab": "GitLabでログインしました",
   "user.settings.security.loginLdap": "AD/LDAPでログインする",
+  "user.settings.security.loginSaml": "GitLabでログインしました",
```

Comment:
sadly we cannot reject a specific message but the whole PR, if you think this PR should be rejected as a whole because of this kind of messages then please tell us :)

My opinion is that the PR can be merged and in a later PR fix the strings with FUZZY messages?

What do you think?